ArjunaJTS/pom.xml

```diff
@@ -53,7 +53,22 @@
         <plugin>
           <groupId>org.codehaus.mojo</groupId>
           <artifactId>exec-maven-plugin</artifactId>
-          <version>1.2</version>
+          <version>3.0.0</version>
+          <configuration>
+           
+          <systemProperty>
```

Comment:
It would be good if these were no longer required. Is the main codebase in that state yet?